diff --git a/71-hori-controllers.rules b/71-hori-controllers.rules
index 8f84240..c27e2e4 100644
--- a/71-hori-controllers.rules
+++ b/71-hori-controllers.rules
@@ -15,3 +15,6 @@ KERNEL=="hidraw*", ATTRS{idVendor}=="0f0d", ATTRS{idProduct}=="0092", MODE="0660
 
 # Hori Nintendo Switch HORIPAD Wired Controller; USB
 KERNEL=="hidraw*", ATTRS{idVendor}=="0f0d", ATTRS{idProduct}=="00c1", MODE="0660", TAG+="uaccess"
+
+# Hori HORIPAD STEAM; USB
+KERNEL=="hidraw*", ATTRS{idVendor}=="0f0d", ATTRS{idProduct}=="01ab", MODE="0660", TAG+="uaccess"

